src/DynamoCore/Configuration/GroupStyleSeparator.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Dynamo.Configuration;
+
+namespace Dynamo.Graph.Annotations
+{
+    /// <summary>
+    /// This class will represent a separator item that once inserted in the list the Style selector will take the MenuItem separator style
+    /// </summary>
+    internal class GroupStyleSeparator : StyleItem
+    {
+    }
```

Comment:
I am still not sure about this but it seems `GroupStyleSeparator` is required to inherit from `StyleItem`? I see no obvious association between them though..

Reply:
I understand, maybe have `GroupStyleSeparator` inherit from `GroupStyleItem` for now?

Reply:
@QilongTang 
If I do the change you are suggesting then **ItemContainerStyleSelector** won't work as expected since it needs to differentiate between **GroupStyleSeparator** class and **GroupStyleItem** class.
Also some other methods are having problems due that the list **ObservableCollection<StyleItem> GroupStyleList** can contain objects of type **GroupStyleSeparator** or **GroupStyleItem** so the next line will gets the wrong items.

//This is supposed to get only items of GroupStyleItem class but also is returning items of GroupStyleSeparator class
**var groupStyleItems = GroupStyleList.OfType<GroupStyleItem>();**

Let me know if you agree or we can discuss it in a call.

Reply:
Ok, I am fine with this